src/Configuration/Dotenv/Parser.hs

```diff
@@ -14,22 +14,24 @@
 
 {-# LANGUAGE CPP #-}
 
-module Configuration.Dotenv.Parse (configParser) where
```

Comment:
Thanks for pointing out to the possible issues that this change might introduce. Agreed, I'll rename back this one.